### PR TITLE
FWI-4477 - Create Kyverno plugin for insights

### PR DESCRIPTION
**Pull request by jslivka**
This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

## Description

This adds a `kyverno` Insights plugin

### What's the goal of this PR?

List all of the `PolicyReport` for each namespace, and also `ClusterPolicyReport`. Collate into a JSON report that can be parsed by Insights to create Action Items.

You can run `main.sh` locally if you modify the `results_file` variable to be something like `./output/kyverno.json` instead of `/output/kyverno.json`

### Output Example

```json
{
  "policyReports": [
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-05T21:21:41Z",
        "generation": 68,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-labels": "500757"
        },
        "name": "cpol-require-labels",
        "namespace": "cert-manager",
        "resourceVersion": "516693",
        "uid": "137179d0-d182-48ca-b5c9-9d50d222394c"
      },
      "results": [
        {
          "message": "validation error: label 'team' is required. rule check-team failed at path /metadata/labels/team/",
          "policy": "require-labels",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "cert-manager-cainjector-56c499cdb8-w42sh",
              "namespace": "cert-manager",
              "uid": "fd47a610-ce2e-4ab6-93b7-c4ad7157c318"
            }
          ],
          "result": "fail",
          "rule": "check-team",
          "scored": true,
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347157
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 9,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-12T14:25:10Z",
        "generation": 17,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-labelsfoo": "500758"
        },
        "name": "cpol-require-labelsfoo",
        "namespace": "cert-manager",
        "resourceVersion": "516694",
        "uid": "a3ac179c-318a-4bd4-a9e6-daa5f2429e9a"
      },
      "results": [
        {
          "message": "validation error: label 'foo' is required. rule check-foo failed at path /metadata/labels/foo/",
          "policy": "require-labelsfoo",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "cert-manager-cainjector-56c499cdb8-w42sh",
              "namespace": "cert-manager",
              "uid": "fd47a610-ce2e-4ab6-93b7-c4ad7157c318"
            }
          ],
          "result": "fail",
          "rule": "check-foo",
          "scored": true,
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347157
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 9,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-12T18:33:51Z",
        "generation": 13,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-pod-probes": "500756"
        },
        "name": "cpol-require-pod-probes",
        "namespace": "cert-manager",
        "resourceVersion": "516692",
        "uid": "6875753a-d442-4ec5-be39-55a3ec9e1854"
      },
      "results": [
        {
          "category": "Best Practices, EKS Best Practices",
          "message": "validation failure: Liveness, readiness, or startup probes are required for all containers.",
          "policy": "require-pod-probes",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "cert-manager-cainjector-56c499cdb8-w42sh",
              "namespace": "cert-manager",
              "uid": "fd47a610-ce2e-4ab6-93b7-c4ad7157c318"
            }
          ],
          "result": "fail",
          "rule": "validate-probes",
          "scored": true,
          "severity": "medium",
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347157
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 4,
        "pass": 2,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-11T15:26:34Z",
        "generation": 32,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-labels": "500757"
        },
        "name": "cpol-require-labels",
        "namespace": "default",
        "resourceVersion": "519115",
        "uid": "302270c2-f047-4c17-8c62-c72ce4cb644c"
      },
      "results": [
        {
          "message": "validation error: label 'team' is required. rule check-team failed at path /metadata/labels/team/",
          "policy": "require-labels",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "nginx-6644c58b8f-hpsrf",
              "namespace": "default",
              "uid": "8bf6ea51-fd82-4b0b-8be5-7df0841dd7ea"
            }
          ],
          "result": "fail",
          "rule": "check-team",
          "scored": true,
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347767
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 4,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-12T14:25:11Z",
        "generation": 19,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-labelsfoo": "500758"
        },
        "name": "cpol-require-labelsfoo",
        "namespace": "default",
        "resourceVersion": "519117",
        "uid": "ddf7c22d-e0a0-4cd4-9ca7-39f1292e5a9f"
      },
      "results": [
        {
          "message": "validation error: label 'foo' is required. rule check-foo failed at path /metadata/labels/foo/",
          "policy": "require-labelsfoo",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "nginx-6644c58b8f-hpsrf",
              "namespace": "default",
              "uid": "8bf6ea51-fd82-4b0b-8be5-7df0841dd7ea"
            }
          ],
          "result": "fail",
          "rule": "check-foo",
          "scored": true,
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347767
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 4,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-12T18:33:51Z",
        "generation": 14,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-pod-probes": "500756"
        },
        "name": "cpol-require-pod-probes",
        "namespace": "default",
        "resourceVersion": "519116",
        "uid": "9972df6a-4e08-440a-ba7e-c51e103110ed"
      },
      "results": [
        {
          "category": "Best Practices, EKS Best Practices",
          "message": "rule passed",
          "policy": "require-pod-probes",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "nginx-6644c58b8f-hpsrf",
              "namespace": "default",
              "uid": "8bf6ea51-fd82-4b0b-8be5-7df0841dd7ea"
            }
          ],
          "result": "pass",
          "rule": "validate-probes",
          "scored": true,
          "severity": "medium",
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347767
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 0,
        "pass": 2,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-06T18:25:12Z",
        "generation": 637,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-labels": "500757"
        },
        "name": "cpol-require-labels",
        "namespace": "insights-agent",
        "resourceVersion": "522379",
        "uid": "84968099-0fd4-4a67-a21a-4deea77315e5"
      },
      "results": [
        {
          "message": "validation error: label 'team' is required. rule check-team failed at path /metadata/labels/team/",
          "policy": "require-labels",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "workloads-28155765-bmpz4",
              "namespace": "insights-agent",
              "uid": "fb149a5f-0593-437a-be77-7d7e5fc1abc6"
            }
          ],
          "result": "fail",
          "rule": "check-team",
          "scored": true,
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689345930
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 56,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-12T14:25:11Z",
        "generation": 225,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-labelsfoo": "500758"
        },
        "name": "cpol-require-labelsfoo",
        "namespace": "insights-agent",
        "resourceVersion": "522380",
        "uid": "d5de3d61-b737-46cf-a2ed-2d9a5300f4ff"
      },
      "results": [
        {
          "message": "validation error: label 'foo' is required. rule check-foo failed at path /metadata/labels/foo/",
          "policy": "require-labelsfoo",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "workloads-28155765-bmpz4",
              "namespace": "insights-agent",
              "uid": "fb149a5f-0593-437a-be77-7d7e5fc1abc6"
            }
          ],
          "result": "fail",
          "rule": "check-foo",
          "scored": true,
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689345930
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 56,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-12T18:33:40Z",
        "generation": 163,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-pod-probes": "500756"
        },
        "name": "cpol-require-pod-probes",
        "namespace": "insights-agent",
        "resourceVersion": "522378",
        "uid": "ad58a092-913e-46bc-ae26-164c6754a7ca"
      },
      "results": [
        {
          "category": "Best Practices, EKS Best Practices",
          "message": "validation failure: Liveness, readiness, or startup probes are required for all containers.",
          "policy": "require-pod-probes",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "workloads-28155765-bmpz4",
              "namespace": "insights-agent",
              "uid": "fb149a5f-0593-437a-be77-7d7e5fc1abc6"
            }
          ],
          "result": "fail",
          "rule": "validate-probes",
          "scored": true,
          "severity": "medium",
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689345930
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 28,
        "pass": 3,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-05T21:21:41Z",
        "generation": 68,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-labels": "500757"
        },
        "name": "cpol-require-labels",
        "namespace": "kube-system",
        "resourceVersion": "516701",
        "uid": "2c6fbb13-0ad6-4b26-9995-4d40dbd611e8"
      },
      "results": [
        {
          "message": "validation error: label 'team' is required. rule check-team failed at path /metadata/labels/team/",
          "policy": "require-labels",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "coredns-5d78c9869d-925lv",
              "namespace": "kube-system",
              "uid": "b4cc3981-9806-4dbb-b770-bc85778c1bec"
            }
          ],
          "result": "fail",
          "rule": "check-team",
          "scored": true,
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347157
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 12,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-12T14:25:11Z",
        "generation": 17,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-labelsfoo": "500758"
        },
        "name": "cpol-require-labelsfoo",
        "namespace": "kube-system",
        "resourceVersion": "516702",
        "uid": "9c6f861d-bf45-47d7-a93c-04a88f11a1d3"
      },
      "results": [
        {
          "message": "validation error: label 'foo' is required. rule check-foo failed at path /metadata/labels/foo/",
          "policy": "require-labelsfoo",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "coredns-5d78c9869d-925lv",
              "namespace": "kube-system",
              "uid": "b4cc3981-9806-4dbb-b770-bc85778c1bec"
            }
          ],
          "result": "fail",
          "rule": "check-foo",
          "scored": true,
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347157
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 12,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-12T18:33:51Z",
        "generation": 13,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-pod-probes": "500756"
        },
        "name": "cpol-require-pod-probes",
        "namespace": "kube-system",
        "resourceVersion": "516703",
        "uid": "abd6b7e3-49c2-4c4c-85ea-06ab1c8f7a6f"
      },
      "results": [
        {
          "category": "Best Practices, EKS Best Practices",
          "message": "rule passed",
          "policy": "require-pod-probes",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "coredns-5d78c9869d-925lv",
              "namespace": "kube-system",
              "uid": "b4cc3981-9806-4dbb-b770-bc85778c1bec"
            }
          ],
          "result": "pass",
          "rule": "validate-probes",
          "scored": true,
          "severity": "medium",
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347157
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 4,
        "pass": 7,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-05T21:21:41Z",
        "generation": 325,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-labels": "500757"
        },
        "name": "cpol-require-labels",
        "namespace": "kyverno",
        "resourceVersion": "522327",
        "uid": "9d4e95e2-6710-4c6a-be40-36e2a0d9a35c"
      },
      "results": [
        {
          "message": "validation error: label 'team' is required. rule check-team failed at path /metadata/labels/team/",
          "policy": "require-labels",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "kyverno-background-controller-bf686c789-hp2vl",
              "namespace": "kyverno",
              "uid": "d9db4109-a6da-40e1-bacb-f7b70e096c69"
            }
          ],
          "result": "fail",
          "rule": "check-team",
          "scored": true,
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347158
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 18,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-12T14:25:11Z",
        "generation": 94,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-labelsfoo": "500758"
        },
        "name": "cpol-require-labelsfoo",
        "namespace": "kyverno",
        "resourceVersion": "522329",
        "uid": "90b44587-e137-45b0-865b-721f31c11047"
      },
      "results": [
        {
          "message": "validation error: label 'foo' is required. rule check-foo failed at path /metadata/labels/foo/",
          "policy": "require-labelsfoo",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "kyverno-background-controller-bf686c789-hp2vl",
              "namespace": "kyverno",
              "uid": "d9db4109-a6da-40e1-bacb-f7b70e096c69"
            }
          ],
          "result": "fail",
          "rule": "check-foo",
          "scored": true,
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347158
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 18,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-12T18:33:51Z",
        "generation": 68,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-pod-probes": "500756"
        },
        "name": "cpol-require-pod-probes",
        "namespace": "kyverno",
        "resourceVersion": "522328",
        "uid": "470a46ee-c319-4e41-bc95-0263b2490105"
      },
      "results": [
        {
          "category": "Best Practices, EKS Best Practices",
          "message": "validation failure: Liveness, readiness, or startup probes are required for all containers.",
          "policy": "require-pod-probes",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "kyverno-background-controller-bf686c789-hp2vl",
              "namespace": "kyverno",
              "uid": "d9db4109-a6da-40e1-bacb-f7b70e096c69"
            }
          ],
          "result": "fail",
          "rule": "validate-probes",
          "scored": true,
          "severity": "medium",
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347158
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 6,
        "pass": 4,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-05T21:21:41Z",
        "generation": 68,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-labels": "500757"
        },
        "name": "cpol-require-labels",
        "namespace": "local-path-storage",
        "resourceVersion": "516709",
        "uid": "9a7882d2-6e83-461d-8c12-820b39462b2c"
      },
      "results": [
        {
          "message": "validation error: label 'team' is required. rule check-team failed at path /metadata/labels/team/",
          "policy": "require-labels",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "local-path-provisioner-6bc4bddd6b-5jhpl",
              "namespace": "local-path-storage",
              "uid": "4d9f134a-7410-49f2-98db-bd5efa2b78b0"
            }
          ],
          "result": "fail",
          "rule": "check-team",
          "scored": true,
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347158
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 3,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-12T14:25:11Z",
        "generation": 16,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-labelsfoo": "500758"
        },
        "name": "cpol-require-labelsfoo",
        "namespace": "local-path-storage",
        "resourceVersion": "516710",
        "uid": "d4ce22e4-a0c2-4031-a3bb-48c226f44599"
      },
      "results": [
        {
          "message": "validation error: label 'foo' is required. rule check-foo failed at path /metadata/labels/foo/",
          "policy": "require-labelsfoo",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "local-path-provisioner-6bc4bddd6b-5jhpl",
              "namespace": "local-path-storage",
              "uid": "4d9f134a-7410-49f2-98db-bd5efa2b78b0"
            }
          ],
          "result": "fail",
          "rule": "check-foo",
          "scored": true,
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347158
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 3,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    },
    {
      "apiVersion": "wgpolicyk8s.io/v1alpha2",
      "kind": "PolicyReport",
      "metadata": {
        "creationTimestamp": "2023-07-12T18:33:51Z",
        "generation": 12,
        "labels": {
          "app.kubernetes.io/managed-by": "kyverno",
          "cpol.kyverno.io/require-pod-probes": "500756"
        },
        "name": "cpol-require-pod-probes",
        "namespace": "local-path-storage",
        "resourceVersion": "516711",
        "uid": "b1cdae66-64de-4b65-9ec9-8564e7b09911"
      },
      "results": [
        {
          "category": "Best Practices, EKS Best Practices",
          "message": "validation failure: Liveness, readiness, or startup probes are required for all containers.",
          "policy": "require-pod-probes",
          "resources": [
            {
              "apiVersion": "v1",
              "kind": "Pod",
              "name": "local-path-provisioner-6bc4bddd6b-5jhpl",
              "namespace": "local-path-storage",
              "uid": "4d9f134a-7410-49f2-98db-bd5efa2b78b0"
            }
          ],
          "result": "fail",
          "rule": "validate-probes",
          "scored": true,
          "severity": "medium",
          "source": "kyverno",
          "timestamp": {
            "nanos": 0,
            "seconds": 1689347158
          }
        }
      ],
      "summary": {
        "error": 0,
        "fail": 2,
        "pass": 0,
        "skip": 0,
        "warn": 0
      }
    }
  ],
  "clusterPolicyReports": []
}
```

### What changes did you make?

Created a simple bash script to retrieve relevant Kyverno report CRDs, with accompanying Dockerfile

### What alternative solution should we consider, if any?

[Internal Ticket FWI-4477](https://fairwinds.myjetbrains.com/youtrack/issue/FWI-4477)

### CODE REVIEW

File: plugins/kyverno/main.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+results_file=/output/kyverno.json
+
+json='{"policyReports":[], "clusterPolicyReports":[]}'
+
+KIND="policyreport"
+
+echo "Retrieving namespaces"
+namespaces=$(kubectl get namespaces -o name)
+IFS=$'\n' namespaces=($namespaces)
+for ns_idx in "${!namespaces[@]}"; do
```
Comment (rbren) [inline plugins/kyverno/main.sh:14]: I think there's a way to just iterate over values
```
for value in "${fruits[@]}"
do
  echo "Value for fruits array is: $value"
done
```

File: plugins/kyverno/main.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+set -e
+trap 'echo "Error on Line: $LINENO"' ERR
+echo "Starting kyverno"
+results_file=/output/kyverno.json
+
+json='{"policyReports":[], "clusterPolicyReports":[]}'
+
+KIND="policyreport"
```
Comment (rbren) [inline plugins/kyverno/main.sh:9]: should we add `clusterpolicyreport`?

File: plugins/kyverno/main.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+echo "Retrieving namespaces"
+namespaces=$(kubectl get namespaces -o name)
+IFS=$'\n' namespaces=($namespaces)
+for ns_idx in "${!namespaces[@]}"; do
+  namespace=${namespaces[$ns_idx]#namespace\/}
+  count=$(kubectl get $KIND -n $namespace -o name | wc -l)
+
+  echo "found $count $KIND for namespace $namespace"
+
+  json="$(jq --argjson list "$(kubectl get $KIND -n $namespace -o json)" '.policyReports += [$list.items]' <<< "$json")"
```
Comment (rbren) [inline plugins/kyverno/main.sh:20]: this is neat! `jq` is magic

File: plugins/kyverno/main.sh (hunk 2 above)
Comment (jslivka) [reply plugins/kyverno/main.sh:9]: It's there on L23

File: plugins/kyverno/main.sh (hunk 1 above)
Comment (jslivka) [reply plugins/kyverno/main.sh:14]: Ah yeah that's cleaner. I cribbed this part from the kubesec script.

**Comment by jslivka** [comment]
Updated to only include the most recent report result for both kinds. I've also included a json schema check like we do for other plugins here.

**Pull request merged**